IMPLEMENTATION_COVERAGE.md

```diff
@@ -171,7 +171,7 @@
 - [ ] update_webhook
 
 ## apigateway
-24% implemented
+25% implemented
```

Comment:
[just thinking out loud] I wonder if there is a way for us to dynamically generate this file... 

I would imagine that we would need to look at what's in botocore and reflect that with the implemented methods for each class. That may or may not be an easy endeavor.